pkg/controller/dmaas_backup_controller.go

```diff
@@ -86,6 +89,95 @@ func NewDMaaSBackupController(
 func (d *dmaasBackupController) processBackup(key string) error {
 	log := d.logger.WithField("key", key)
 
-	log.Infof("processing backup")
+	log.Debug("Processing dmaasbackup")
+
+	ns, name, err := cache.SplitMetaNamespaceKey(key)
+	if err != nil {
+		log.WithError(err).Errorf("failed to split key")
+		return nil
+	}
+
+	original, err := d.lister.DMaaSBackups(ns).Get(name)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		return errors.Wrapf(err, "failed to get dmaasbackup")
+	}
+
+	switch original.Spec.State {
+	case v1alpha1.DMaaSBackupStateEmpty, v1alpha1.DMaaSBackupStateActive:
+		// process only active/new dmaasbackup
+		if original.Status.Phase == v1alpha1.DMaaSBackupPhaseCompleted {
+			log.Debug("DMaaSBackup completed, skipping")
```

Comment:
is there a way to increase sync time of such objects, so that, it won't come again and again?

Reply:
or even better is to remove it from informer so that event won't come

Reply:
This will only come on update. We can move this check to NewDMaaSBackupController.